Problem4/test/problem4/Problem4Test.java

```diff
@@ -0,0 +1,29 @@
+/*
+ * To change this license header, choose License Headers in Project Properties.
+ * To change this template file, choose Tools | Templates
+ * and open the template in the editor.
+ */
+package problem4;
+
+import java.util.Arrays;
+import static org.junit.Assert.*;
+import org.junit.Test;
+
+/**
+ *
+ * @author manxboy
+ */
+public class Problem4Test {
+    
+    @Test
+    public void testSort() {
```

Comment:
Your testing should include edge cases like empty arrays and arrays of a single element.